doc/sphinx/source/programmable/index.rst

```diff
@@ -12,10 +12,12 @@ Backends are responsible for function execution and value allocation. They
 can be used to :doc:`carry out a programmed computation<../howto/execute>`
 from a framework by using a CPU or GPU; or they can be used with an *Interpreter* 
 mode, which is primarily intended for testing, to analyze a program, or for a 
-framework developer to develop a custom UI or API. 
+framework developer to develop a custom UI or API. Other experimental APIs 
```

Comment:
Okay, clarified it as requested.  Seem(s/ed) we might have written previously on the importance of frameworks owning their UI so this may have been seemingly relevant.